Probesys/bileto
# new: Allow to change the priority of the tickets

Related to https://github.com/Probesys/bileto/issues/11

Changes proposed in this pull request:

- Add a button next to the "priority" section to change it
- Add a form to edit urgency / impact / priority
- Add automatic calculation of the priority based on the urgency and impact values

How to test the feature manually:

1. Open a ticket, click on the "edit" button of the priority
2. Change the impact and urgency, check the priority value is changed automatically
3. Optionally change the priority value
4. Save the form and check the urgency / impact / priority have changed on the ticket page

Pull request checklist:


_If you think one of the item isn’t applicable to the PR, please check it
anyway and precise `N/A` next to it._
